realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -956,15 +927,12 @@ boolean contains(Class<? extends RealmModel> clazz) {
     /**
      * Adds a change listener to the Realm.
      * <p>
-     * The listeners will be executed:
-     * <ul>
-     * <li>Immediately if a change was committed by the local thread</li>
-     * <li>On every loop of a Handler thread if changes were committed by another thread</li>
-     * <li>On every call to {@link io.realm.Realm#refresh()}</li>
-     * </ul>
+     * The listeners will be executed on every loop of a Handler thread if
+     * the current thread or other threads committed changes to the Realm.
      *
-     * Listeners are stored as a strong reference, you need to remove the added listeners using {@link #removeChangeListener(RealmChangeListener)}
-     * or {@link #removeAllChangeListeners()} which removes all listeners including the ones added via anonymous classes.
+     * Realm instances are thread singletons and cached, so listeners should be
+     * removed manually even if calling {@link #close()}, otherwise there is a
```

Comment:
I think we have two options to use `otherwise`.

1. `{@link #close()}. Otherwise,`.
2. `{@link #close()}; otherwise`.

Two consecutive commas make me a bit wonder.